Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I click on the menu zone "Platform"
+        And I click on the "Content structure" link
+        And I click on the side menu option "Create a content"
+        And I click on the content type "Author"
+        And I fill in "Name" with "AuthorTest"
```

Comment:
We could shorten the sentences:
"
        And I click on the menu zone "Platform"
        And I click on the "Content structure" link
        And I click on the side menu option "Create a content"
        And I click on the content type "Author"
        And I fill in "Name" with "AuthorTest"
"

With 
        And I create a content of content type "Author" with
            | name       | author | email |
            | AuthorTest |        |       | 

Where the creation of the content would follow all the steps described before.